fix misspells and minor cleanup (#44)
squid-cache/squid-cache.github.io: docs/SquidFaq/OperatingSquid.md

only return TCP_IMS_MISS if some other client causes a newer version
of the object to be pulled into the cache.

> ## Why do I need to run Squid as root? why cannot I just use cache_effective_user root?

Reviewer: contraction changed grammar
```suggestion
## Why do I need to run Squid as root? why can I not use cache_effective_user root?
```
